mobile/mobile_46.asm

```diff
@@ -1569,7 +1569,9 @@ asm_118d9f:
 	ld a, [wcd38]
 	and a
 	jr nz, .asm_118db2
-	ld a, $8f
+	ld a, $8f ; DION addr $1e + request $8 + Name $5
```

Comment:
I know it might feel a little stupid to do now there's only one use, but please make constants for all of these, and use the few existing constants that already exist for `NAME_LENGTH_JAPANESE` and such. They'll come in handy later.

Reply:
It's used in quite a few places, wanted to find the layout of the request part again before doing that
That comment was originally written in 2020 and i cannot remember the gts requests worked now

Reply:
Then make a constant for it and comment it there. I can smell from a mile that this comment will stay here for ages even after a proper constant is used for it.

Reply:
[d3a016b](https://github.com/pret/pokecrystal/pull/993/commits/d3a016b7906eeb88183fbfa092d0b1ddf4e5fdef)